README.md

```diff
@@ -14,12 +14,41 @@ Please see [CONTRIBUTING.md](./CONTRIBUTING.md).
 
 ## Initialization
 
+The Tracer defaults to sending spans over UDP to the jaeger-agent running on localhost.
+
+```javascript
+var initTracer = require('jaeger-client').initTracer;
+
+// See schema https://github.com/jaegertracing/jaeger-client-node/blob/master/src/configuration.js#L37
+var config = {
+  serviceName: 'my-awesome-service',
+};
+var options = {
+  tags: {
+    'my-awesome-service.version': '1.1.2',
+  },
+  metrics: metrics,
+  logger: logger,
+};
+var tracer = initTracer(config, options);
+```
+
+#### Reporting spans via HTTP
```

Comment:
I would add some explanation why someone might want that, and also clarify that in this case the data is not going through the agent.